src/include/distributed/remote_transaction.h

```diff
@@ -106,4 +119,9 @@ extern void CoordinatedRemoteTransactionsPrepare(void);
 extern void CoordinatedRemoteTransactionsCommit(void);
 extern void CoordinatedRemoteTransactionsAbort(void);
 
+/* remote savepoint commands */
+extern void RemoteTransactionsSavepointBegin(SubTransactionId subId);
```

Comment:
I think it'd be better to stay in line with the naming from the functions just above.